### DISCUSSION TITLE
astra UI v0.0.2-alpha

### DISCUSSION THREAD

**Discussion by AstraThreshold** [AstraThreshold/oled-ui-astra#1] [Announcements]
#### Features
+ Fixed some issues.
+ Modified the speed of some animations.
+ Added transition animation to selector.
+ Added a way to turn pages: scrolling.
---
#### 特性
+ 修复了一些问题。
+ 调整了某些动画的速度。
+ 为`selector` 加入了过渡动画。
+ 增加了一种页面切换方式：滚动。
---
### 循此苦旅，直抵群星。
---
**Full Changelog**: https://github.com/dcfsswindy/oled-ui-astra/compare/v0.0.1-alpha...v0.0.2-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/dcfsswindy/oled-ui-astra/releases/tag/v0.0.2-alpha'>astra UI v0.0.2-alpha</a>.</em>

**Comment by leisure9527** [comment]
怎么实现按钮的交互

**Reply by ghost** [reply]
催更啊！！！挺喜欢这个的

**Reply by ghost** [reply]
成功了，自己写个定时器扫描按键，调用slector->goNext()等等就行
